Client/core/CConsole.cpp

```diff
@@ -543,10 +543,43 @@ void CConsole::FlushPendingAdd ( void )
         m_strPendingAdd = "";
 
         // Trim new buffer
-        if ( strBuffer.length () > CONSOLE_SIZE )
+        uint uiBufferLength = strBuffer.length ();

```

Comment:
Just a small thing for the future:
Using `std::size_t` would be better for the best compatibility between 32-bit and 64-bit systems (`sizeof(size_t)` is 4 bytes on a 32-bit system and 8 bytes on a 64-bit system).

However, since the MTA client is 32-bit only it's not an issue here.